components/molecules/FeaturedHighlightsPanel/featured-highlights-panel.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import React from "react";
+
+interface FeaturedHighlightsPanelProps {
+  highlightIds: { id: number; title: string }[];
+}
+const FeaturedHighlightsPanel = ({ highlightIds }: FeaturedHighlightsPanelProps) => {
+  return (
+    <div className="border max-w-[299px] rounded-lg p-5 space-y-4">
```

Comment:
Why the `[299px]`? Is this width not able to be standardized? I assume that this width will be similar to other components and we should adhere or create the standard.

Reply:
tailwind has no utility value for `229px` in max-w, But converting the px to rem value should be the standard we already have in place... I'll do that shortly 👍